services/slam/builtin/builtin.go

```diff
@@ -197,7 +197,7 @@ func runtimeServiceValidation(
 	camStreams []gostream.VideoStream,
 	slamSvc *builtIn,
 ) error {
-	if len(cams) == 0 {
+	if !slamSvc.useLiveData {
```

Comment:
We may want to consider allowing yaml files to be generated even when `use_live_data == false`, but the sensor list exists

Reply:
`use_live_data` is a mandatory parameter, which means a yaml file should only be generated when `use_live_data == true`.

Reply:
also worth noting any yaml generation behavior changes would be out of scope for this ticket imo

Reply:
I remembered us discussing the need to copy over yaml files between directories if running in offline mode which is why i brought this up. I agree it is probably out of scope for this ticket

Reply:
yeah, currently to run orbslam in offline mode you are required to have a yaml file, which can be tedious to manage